node/src/container_chain_spawner.rs

```diff
@@ -291,35 +287,22 @@ impl ContainerChainSpawner {
             container_chain_cli_config.database.set_path(&db_path);
 
             // Start container chain node
-            let (
-                mut container_chain_task_manager,
-                container_chain_client,
-                container_chain_db,
-                collate_on,
-            ) = start_node_impl_container(
-                container_chain_cli_config,
-                orchestrator_client.clone(),
-                relay_chain_interface.clone(),
-                orchestrator_chain_interface.clone(),
-                collator_key.clone(),
-                sync_keystore.clone(),
-                container_chain_para_id,
-                orchestrator_para_id,
-                validator,
-            )
-            .await?;
+            let (mut container_chain_task_manager, container_chain_client, container_chain_db) =
+                start_node_impl_container(
+                    container_chain_cli_config,
+                    orchestrator_client.clone(),
+                    relay_chain_interface.clone(),
+                    orchestrator_chain_interface.clone(),
+                    collator_key.clone(),
+                    sync_keystore.clone(),
+                    container_chain_para_id,
+                    orchestrator_para_id,
+                    validator && start_collation,
```

Comment:
Previously when we wanted to sync a container chain we started it with "collator = true", but we didn't call the `start_consensus_container` part. Now we simply start it with "collator = false". The only difference is that we no longer call `start_relay_chain_tasks`, which were not needed before when not running as collator.